docs/user/status.md

```diff
@@ -0,0 +1,108 @@
+# Conditions
+
+[The ClusterVersion object](../dev/clusterversion.md) sets `conditions` describing the state of the cluster-version operator (CVO).
+This document describes those conditions and, where appropriate, suggests possible mitigations.
+
+## RetrievedUpdates
+
+When `RetrievedUpdates` is `True`, the CVO is succesfully retrieving updates, which is good.
+When `RetrievedUpdates` is `False`, `reason` will be set to explain why, as discussed in the following subsections.
+In all cases, the impact is that the cluster will not be able to retrieve recommended updates, so cluster admins will need to monitor for available updates on their own or risk falling behind on security or other bugfixes.
+
+### NoUpstream
+
+No `upstream` server has been set to retrieve updates.
+
+Fix by setting `spec.upstream` in ClusterVersion to point to a [Cincinnati][] server, for example https://api.openshift.com/api/upgrades_info/v1/graph .
+
+### InvalidURI
+
+The configured `upstream` URI is not valid.
+
+Fix by setting `spec.upstream` in ClusterVersion to point to a valid [Cincinnati][] URI, for example https://api.openshift.com/api/upgrades_info/v1/graph .
+
+### InvalidID
+
+The configured `clusterID` is not a valid UUID.
+
+Fix by setting `spec.clusterID` to a valid UUID.
+The UUID should be unique to a given cluster, because it is the default value used for reporting Telemetry and Insights.
+It may also be used by the CVO when making Cincinnati requests, so that Cincinnati can return update recommentations tailored to the specific cluster.
+
+### NoArchitecture
+
+The set of architectures has not been configured.
```

Comment:
Why doesn't this "reason" have an explanation of how to fix? This is the only one that does not so it should have something here.

Reply:
> Why doesn't this "reason" have an explanation of how to fix?

I forgot ;).  Added the usual "CVO bug, need an update" boilerplate with c79a2e4 -> 69c81b9.